docs/api/system-preferences.md

```diff
@@ -371,11 +371,11 @@ maps to [NSApplication.effectiveAppearance](https://developer.apple.com/document
 
 Please note that until Electron is built targeting the 10.14 SDK, your application's
```

Comment:
As of Electron 8 we are targetting this SDK so this entire paragraph can be removed probably

Reply:
I was wondering about that...I'll remove.

Reply:
https://github.com/electron/electron/pull/22489/commits/a97310c1815a4728556c05c4407286c54798f54a